packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -60,14 +65,17 @@ const WalletButton: React.FC<WalletButtonProps> = ({
         }
     };
 
-    const buttonFontSizeMapper: Record<Extract<TGenericSizes, 'lg' | 'md' | 'sm'>, TGenericSizes> = {
+    const buttonFontSizeMapper: Record<
+        Extract<TGenericSizes, 'lg' | 'md' | 'sm'>,
+        Extract<TGenericSizes, 'sm' | 'xs'>
+    > = {
         lg: 'sm',
         md: 'sm',
         sm: 'xs' as const,
```

Comment:
@aizad-deriv looks like just property `sm` as const

Reply:
Yes, put it after closing bracket